src/lib/routes/admin-api/api-token.ts

```diff
@@ -64,33 +64,6 @@ export const tokenTypeToCreatePermission: (tokenType: ApiTokenType) => string =
         }
     };
 
-const permissionToTokenType: (permission: string) => ApiTokenType | undefined =
-    (permission) => {
-        if (
-            [
-                CREATE_FRONTEND_API_TOKEN,
-                READ_FRONTEND_API_TOKEN,
-                DELETE_FRONTEND_API_TOKEN,
-                UPDATE_FRONTEND_API_TOKEN,
-            ].includes(permission)
-        ) {
-            return ApiTokenType.FRONTEND;
-        } else if (
-            [
-                CREATE_CLIENT_API_TOKEN,
-                READ_CLIENT_API_TOKEN,
-                DELETE_CLIENT_API_TOKEN,
-                UPDATE_CLIENT_API_TOKEN,
-            ].includes(permission)
-        ) {
-            return ApiTokenType.CLIENT;
-        } else if (ADMIN === permission) {
-            return ApiTokenType.ADMIN;
-        } else {
-            return undefined;
-        }
-    };
-
```

Comment:
I think we're losing something here. 

Previously we said that anyone with any of the CRUD permissions for that token could read them, but now we specifically require the READ permission.

Reply:
Good catch! I completely overlooked this! Added them back in the last commit